src/views/forms/FormPersistentDelegationRequestTransaction/FormPersistentDelegationRequestTransactionTs.ts

```diff
@@ -92,10 +92,9 @@ import { feesConfig as defaultFeesConfig, networkConfig } from '@/config';
 export enum HarvestingAction {
     START = 1,
     STOP = 2,
-    SWAP = 3,
-    ACTIVATE = 4,
-    SINGLE_KEY = 5,
+    SINGLE_KEY = 3,
```

Comment:
are you missing tests for these changes?

Reply:
Yes, it will be handled in the following PRs. I had attached the following note in the description.
> Unit tests for the FormPersistentDelegationRequestTransaction component will be added in another PR.